src/sidebar/components/sidebar-content-error.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+/**
+ * An error message to display in the sidebar.
+ */
+function SidebarContentError({ errorMessage, errorDetails, onLogin, auth }) {
+  return (
+    <Fragment>
+      <div className="annotation-unavailable-message">
+        <div className="annotation-unavailable-message__icon" />
+        <p className="annotation-unavailable-message__label">
+          {auth.status === 'logged-out' ? (
+            <span>
```

Comment:
The `<span>` here isn't needed as far as the DOM structure is concerned, but you do need some kind of container for the child elements. This is where `Fragment` is useful, it provides a container in JSX which does not appear in the DOM.